infra/modules/aws-psoxy-instance/main.tf

```diff
@@ -19,6 +19,8 @@ resource "aws_apigatewayv2_integration" "map" {
   integration_uri           = aws_lambda_function.psoxy-instance.invoke_arn
   request_parameters        = {}
   request_templates         = {}
+  payload_format_version    = "2.0"
```

Comment:
This is the main change. Gateway has two payload formats, by default we were using 1.0 and I was having a lot of trouble with that version. Switched to 2.0 and could use directly the classes in their library.

Reply:
ah, OK ... yeah, I was confused as to why those classes didn't match examples in AWS console / their documentation.